Documentation/api-reference.md

```diff
@@ -385,8 +385,9 @@ For more information, please read an overview of the [AEP Identity Service](http
   }
```

Comment:
One quick comment for the prerequisites section, can you rephrase `Refer our Getting Started Guide<link>` to `Refer to the Getting Started Guide<link>.`

Reply:
Refer to the ..